INSTRUCTION.md

```diff
@@ -0,0 +1,17 @@
+# Instructions for Running the ToDo App Docker Container
+
+## 1. Docker Hub Repository Link
+
+You can find the Docker image for the ToDo app in my personal Docker Hub repository here:
+
+[My Docker Hub Repository - todoapp](https://hub.docker.com/r/srtrace/py-todo-app)
+
+## 2. Build the Docker Image
+
+## Build the Image
+
+To build the Docker image, run:
+
+```bash
+docker build -t docker build -t srtrace/py-todo-app:1.0.0 .
```

Comment:
The Docker build command is duplicated. It should be corrected to `docker build -t srtrace/py-todo-app:1.0.0 .` to ensure the image is built correctly.